### PR TITLE
feat: <Dropdown> - ref forwarding

### PR DESCRIPTION
**Pull request by SergeyRoyt**
https://monday.monday.com/boards/3532714909/pulses/4408947808

### CODE REVIEW

**Review by shlomitc — COMMENTED** [review]
I suggest adding a test, just to make sure the functionality works and its another way to explain how to use it

File: src/components/Dropdown/__stories__/dropdown.stories.scss [hunk 1]
```diff
@@ -37,4 +37,8 @@
     align-items: center;
     justify-content: center;
   }
+
+  &_button {
```
Comment (shlomitc) [inline src/components/Dropdown/__stories__/dropdown.stories.scss:41]: My two cents about our code examples - they should be copy-pastable into the user's code.
In that case, className is not something that can be easily copied.
I'd suggest using `Flex`'s `gap` and maybe `Box` if it's something that's really necessary.
We can also wrap with our internal example presentation components that know how to arrange elements as a proper example. Then, when the user will copy the example, he can simply delete it.

File: src/components/Dropdown/__tests__/driver.js [hunk 2]
```diff
@@ -38,6 +38,10 @@ export default class DropdownDriver {
     return this.renderResult.asFragment();
   }
 
+  get dropdownWrapper() {
```
Comment (shlomitc) [inline src/components/Dropdown/__tests__/driver.js:41]: I'm not sure this is in use in your current PR changes, so I wonder if you need it

File: src/components/Dropdown/__stories__/dropdown.stories.scss (hunk 1 above)
Comment (SergeyRoyt) [reply src/components/Dropdown/__stories__/dropdown.stories.scss:41]: Yeah, but this is not really an example we expect user to copy

File: src/components/Dropdown/__tests__/driver.js (hunk 2 above)
Comment (SergeyRoyt) [reply src/components/Dropdown/__tests__/driver.js:41]: Removing it - eventually not needed it

**Review by shlomitc — APPROVED** [review]

**Pull request merged**